main.py

```diff
@@ -0,0 +1,4 @@
+import Telegramsender
+import binanceinfo
+if __name__=='__main__':
+    binanceinfo,Telegramsender
```

Comment:
Totaly incorrect way to call the function from imported module. For example, after `binanceinfo` type `.` & see recomendations, if not showed automatically, press `CRTL + Space`